lib/maths/CTimeSeriesModel.cc

```diff
@@ -387,10 +399,15 @@ class CTimeSeriesAnomalyModel {
 
 private:
     //! Update the anomaly model with a sample of the current feature vector.
-    void sample(const CModelProbabilityParams& params, core_t::TTime time, double weight);
+    void sample(const CModelProbabilityParams& params, double weight);
 
     //! Compute the probability of the anomaly feature vector.
-    bool anomalyProbability(core_t::TTime time, double& result) const;
+    bool anomalyProbability(double& result) const;
+
+    //! Get the largest probability the model counts as anomalous.
+    double largestAnomalyProbability() const {
```

Comment:
A bit of a nit pick - feel free to disregard! - Rename this method to say `twiceLargestSignificantProbability`?

Reply:
I think I'll leave this. It is the probability which is deemed sufficiently anomalous to update this model and although it is currently twice the cutoff probability to output a result there is nothing that requires this, it was just a reasonable value.